Fix #1273 - Update GeoLite2-City extraction for new directory structure

acmiyaguchi opened the pull request:
This fixes #1273. The GeoLite2-Cirty archive now includes several additional files that caused the `download-geolite` script to generate an invalid mmdb binary. This fixes issues with the `GeoCityLookup` step of the beam pipeline when running the decoder locally for integration testing purposes.

scholtzan commented on ingestion-beam/bin/download-geolite2:
```diff
@@ -6,13 +6,27 @@
 # from MaxMind for use in development.
 BASE_DIR="$(dirname "$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )")"
 DOWNLOAD_URL="https://download.maxmind.com/app/geoip_download?edition_id=GeoLite2-City&license_key=${MM_LICENSE_KEY}&suffix=tar.gz"
```
I wonder if it would make sense to have a check for whether `MM_LICENSE_KEY` is set in the script. Right now, even if it's not set it'll still try to download the files without erroring out. 

Having a error message printed out might make it easier for people setting up the project for the first time to figure out what's going wrong. Though there is documentation somewhere for this, so probably not super urgent.

acmiyaguchi commented:
Added in a small check for the license, and fixed a bug that would keep redownloading the file even when it existed locally. I tested this in the following way:

* `unset MM_LICENSE_KEY`, remove existing mmdb, and run to failure
* `export MM_LICENSE_KEY=...` and check for mmdb
* rerun and check that it terminates quickly
